---
title: "Ribosome accumulation during early phase resistance training"
author: Daniel Hammarström^1,2,£^&ast;,Sjur J. Øfsteng^2,£^, Nicolai B. Jacobsen^1^,
  Krister B. Flobergseter^1^,  Bent R. Rønnestad^1^, Stian Ellefsen^1,3^
date: ''
output:
  word_document:
    reference_docx: resources/manuscript_word_template.docx
csl: resources/experimental-physiology.csl
bibliography: resources/bibliography.bib
header-includes: \usepackage{wasysym}
editor_options:
  chunk_output_type: console
---
<!-- This is a comment  -->
<!-- Add comments after each reference for tracking (eg[@RN2364]<!-- comment -->
<!-- After references, add a row break without space (this will put the reference on the end on the line). -->

<!-- The formatting of the manuscript is based on faseb journal, find csl-files at https://github.com/citation-style-language/styles -->

<!-- Figure texts are included in the end of the manuscript file -->
<!-- Tables in a separate file -->

```{r, include = FALSE}

source("./R/libs.R")
# Settings for chunk options
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)

```


^1^ Section for Health and Exercise Physiology, Department of Public Health and Sport Sciences, Inland Norway University of Applied Sciences, Lillehammer, Norway. 

^2^ Swedish School of Sport and Health Sciences, Stockholm, Sweden.

^3^ Innlandet Hospital Trust, Lillehammer, Norway. </div>

^£^ These authors contributed equally to this work




<div custom-style="correspondence_style">
&ast; Correspondance

Daniel Hammarström, Section for Health and Exercise Physiology, Department of Public Health and Sport Sciences, Inland Norway University of Applied Sciences, Lillehammer, Norway. Email: daniel.hammarstrom@inn.no  </div>



<div custom-style="runningtitle_style">**Running title:** Ribosome biogenesis in early phase resistace training</div>


\pagebreak


# Abstract
<div custom-style="abstract_style">
<!-- Word limit 200 -->
Abstract...


**Keywords:** Resistance training, ribosome biogenesis


</div>

\pagebreak

# Introduction

Skeletal muscle is a critical target for interventions that promotes health across the lifespan [@RN763]<!-- Wolfe 2006 -->,
with resistance training (RT) being the advocated remedy.  
RT promotes changes in the balance between muscular protein breakdown and synthesis through repeated episodes of elevated post-exercise protein synthesis, leading to net positive protein balance over time. Indeed, one bout of resistance exercise acutely increases skeletal muscle protein synthesis for up to 48 hrs after the exercise [@RN786],<!-- Phillips 1997 -->
with subsequent repeated bouts leading to accumulation of muscle protein over time
[@RN1272<!-- Phillips review 2014 -->;
@RN1642]<!-- Brook 2015 experiment -->. 
In recent years, this view has been supplemented<!-- supported? --> by evidence suggesting that chronic RT leads to increased resting synthetic rate of muscle protein
[@RN1521;<!-- Kim 2005 Fasted state skeletal muscle PS altered with training  --> 
@RN1866;<!-- Wilkingson 2008 -->
@RN1897],<!-- Reidy 2017 Increased rested state protein synthesis -->
which has been postulated to be associated with increased translational capacity, i.e. accumulation of ribosomes
[@RN1897;<!-- Reidy 2017 -->
@RN1912]<!-- Figueiredo 2017 Ribosome biogenesis review -->.
This notion is supported by exercise-induced increases in total RNA, a proxy marker of ribosome abundance, occuring in proportion to muscle hypertrophy
[@RN1644;<!-- Figueiredo 2015 Correlation between total RNA fold change and CSA hyp -->
@RN1755; <!-- Stec Ribosome bio higher in high responders -->
@RN2358]<!-- Volume dependent RNA accum relates to volume depend hyper -->
and the close relationship between RNA abundance and protein synthesis
[@RN1754;<!-- West 2016 correlation between total RNA and protein synthesis in C2C12-->
@RN2145].<!-- Millward 1973 relationship between RNA and protein synthesis in rat in rats in vivo constant infusion -->
Conversely, inhibition of ribosomal RNA (rRNA) transcription and inhibition of its up-stream transcription factors act to diminish muscle cell growth
[@RN1755<!-- Stec 2016 Pol I-specific inhibition leads to GF induced myotube hypertrophty -->;
@RN1754<!-- West 2016 Inhibition of c-Myc (through inibitor) reduces RNA and MPS-->;
@RN1810]<!-- von Walden 2016 Inhibition of Pol 1 rDNA tracsription and myotube growth -->.  

Biosynthesis of novel ribosomes is a complex, highly coordinated and energy demanding process that involves synthesis of both proteins and four different mature rRNA
[@RN1920;<!-- Warner review yeast -->
@RN1820;<!-- Moss review -->
@RN1810]<!-- Inhibition of Pol II affects trasncription at Pol I -->.
This process is limited at the level of pre-rRNA transcription by RNA polymerase I (Pol I) which in turn is regulated by coordinated assembly of a complex of transcription factors at the rDNA promoter
[@RN1820]<!-- Moss review -->.
Specifically, phosphorylation of the upstream bindning factor (UBF) is needed to initiate transcription
[@RN2582; <!-- Tuan UBF phosphorylation important for SL1-UBF interaction -->
@RN2563]<!--Lin 2007 phospho of UBF shows it has a role in recruioting SL1 to the rDNA promoter -->
and this is related to signaling through the mechanosensitive mTOR pathway as its inhibition blocks UBF phosphorylation and subsequent rRNA transcription
[@RN1632; <!-- Nader rapamycin block UBF availability  -->
@RN2564].<!-- Hannan mTOR  -->
Interestingly the availability of UBF *per se* has been shown to also regulate rRNA transcription
[@RN2566]<!-- Hannan 1996 over expression of UBF leads to rRNA transcription in cardiomyocytes -->
through control of rDNA gene activity
[@RN2556].<!-- Sanij, UBF depleting leads to inactivation of rDNA genes -->

Resistance exercise is a potent stimuli for rRNA transcription as a single session leads to increases in pre-rRNA
[@RN1656<!-- A single session upregulates 45 S 24 h post ex -->;
@RN1037<!-- Nader 2014 45S up in response to 4 h after acute exercise -->]
and repeated bouts lead to accumulation of mature rRNA thus also total RNA and presumably functional ribosomes
[@RN1656;<!-- Stec2015 A single session ns ~ 9% increase in young, -4% in old -->
@RN1520;<!-- Bickel 2005 Two bouts electric stimulation, young ~ 30% incr-->
@RN2358; <!-- Four sessions ~29 and 18 % in mod vs. low volume|31 sessions 18/12% young-->
@RN1755<!-- Stec 2016 12 sessions 26, 9 and 6% increase in high, mod and low responders OLD-->;
@RN1644;<!-- Figuieredo 2015 16 sessions 49% increase young -->
@RN1809; <!-- Brook 18 sessions 8 and 40% (9 sessions) and 15,22% (18 sessions) old and young-->
@RN1897<!-- 11% 36 sessions young -->].
However, the true time course of ribosomal transcription and accumulation in response to RT remains largely unstudied, with only a few studies having investigated exercise-induced changes in rRNA over multiple time-points.
As such, Bickel *et al*. showed that two consecutive bouts of electrically induced muscle contractions were associated with increased levels of total RNA, with peak values being observed 72 hrs after the second bout
[@RN1520]<!-- Total RNA increase 72 h after a second bout -->. 
Brook *et al.*, reported peak values after nine sessions followed by a slight decrease to after 18 sessions of RT
[@RN1809],
resembling data from our lab where five sessions of RT led to marked increase in total RNA levels (per-unit muscle tissue), whereupon a numerical lowering occurred to after the last training session of the 12 wk interventions (31 sessions).
[@RN2358].
Interestingly, total RNA accumulation in the initial phase was shown to be training volume sensitive as three sets per exercise in leg exercises led to increased total RNA and rRNA accumulation compared to a single set per exercise, coinciding with differences in muscle hypertrophy after 12 weeks of RT.
[@RN2358].
These data suggest that ribosome accumulation reaches a plateau in the early phase of RT and that increases are sensitive to training volume in constant volume protocols. 

Based on these observations we hypothesize that (1) ribosome accumulation occurs during the early phase (3-4 weeks) of RT, within which this accumulation (2) reaches a plateau when RT volume is kept constant, (3) displays fluctuations in response to fluctuating training volume and (4) is partially reversed to one week after cessation of RT. In addition to addressing these hypotheses we aimed to relate RNA accumulation to total UBF levels and muscle growth.

# Methods

## Study overview

```{r length-of-control-period}

# Calculate the average of the control period (in weeks) for the CTRL group
ctrl_days <- read_excel("./data/tissue/tr010_tissue.xlsx") %>%
  filter(participant %in% paste0("P", 9:15)) %>%
  mutate(date = as.Date(date, origin = "1899-12-30")) %>%
  group_by(participant, time) %>%
  summarise(date = mean(date)) %>%
  pivot_wider(names_from = time, values_from = date) %>%
  mutate(ctrl.period = postctrl - S0) %>%
  ungroup() %>%
  summarise(m = round(as.numeric(mean(ctrl.period))/7,1), 
            s = round(as.numeric(sd(ctrl.period))/7, 1), 
            min = round(as.numeric(min(ctrl.period))/7, 1), 
            max =  round(as.numeric(max(ctrl.period))/7, 1))
  

ctrl <- paste0(ctrl_days[1,1], " (", ctrl_days[1,2], ")") 
```

Eighteen volunteers were recruited to the study. Eligible participants were non-smokers between 18 and 35 years of age with a training history of less than one RT session per week during the six months leading up to the study. Exclusion criteria were consumption of dietary supplements or medication with known effects on muscle metabolism, injuries causing impaired strength and/or affecting their ability to perform RT, symptoms or history of disease, and known adverse reactions to local anesthetics. Participants were allocated to either an training group (TRAIN, *n* = 11) or a non-training control group (CTRL, *n* = 8; see Table 1 for participant characteristics; see Figure 1A for overview of the intervention). TRAIN performed a 12 session RT protocol lasting for 3-4 weeks, consisting of 10 repetition maximum (RM) unilateral knee-extension, with the two legs conducting RT with different volume profiles, allowing within-participant comparison of the effects of volume regimes. In TRAIN, one leg conducted RT with constant volume throughout the intervention (CONST, 6 sets per session) and the other leg performed RT with variable volume (VAR, 3 blocks of four sessions with 6, 3 and 9 sets per session, respectively; Figure 1A). CTRL did not partake in RT and were instructed to continue their everyday activities. Muscle biopsies were sampled bilaterally in TRAIN before and 48 hrs after the first session, as well as 48 hrs after the fourth, fifth, eight, ninth and twelfth session, and after a seven days of de-training. Muscle biopsies were obtained from CTRL at three occasions; at baseline and 48 hrs and 3-5 weeks (average (SD) `r ctrl`) after the first sampling event. TRAIN and CTRL performed strength assessments > seven days prior to the first biopsy sampling (TRAIN; CTRL), 72 hrs after the twelfth session (TRAIN) and 24 h after the last biopsy (TRAIN, following de-training; CTRL). Appendicular lean mass (Dual-energy X-ray absorptiometry, DXA) and muscle thickness of *m. vastus lateralis* were assessed prior to the first biopsy (TRAIN and CTRL) as wells as before the second to last (TRAIN) and last (TRAIN and CTRL) biopsy.

## RT protocol
Prior to all RT sessions, participants performed a standardized warm-up consisting of 5 min ergometer cycling (rating of perceived exertion (RPE): 12-14), followed by ten repetitions of push-ups, sit-ups and back-extensions. After warm-up, participants performed unilateral knee-extension with the prescribed number of sets. Each set was prescribed with 10 repetitions maximum (RM). When sets were completed with either fewer (8) or more (12) repetitions, the resistance was adjusted accordingly. Inter-set rest periods were 90 sec. Throughout the intervention RT sessions were alternatingly initiated by training the right and left leg, changing every other session, with the contralateral leg being trained in the rest period between sets of the first leg. The second session of each four-session block (session 2, 6 and 10) was performed at a sub-maximal resistance (~90% of the previous session) with the same number of repetitions (10). Within each session, participants also conducted two sets of three upper-body exercises (bench press, lateral pull-down and shoulder press; 10RM).<!--Prior to each session, participants were asked to rate muscle soreness on a scale from 0 to 10 (0 = no soreness, 10 = hurting very bad, e.g. not being able to walk properly) [@Vickers2010]. After each session, participants were asked to rate their perceived exertion (Borg CR10).-->
After completion of each session participants were given a standardized drink to aid recovery (0.15 g kg<sup>-1</sup> protein, 11.2 g kg<sup>-1</sup> carbohydrates and 0.5 g kg<sup>-1</sup> fat).


## Muscle strength, body composition and muscle thickness assessments
Muscle strength was assessed as maximal voluntary isokinetic (90&deg; sec<sup>-1</sup>) and isometric (60&deg; angle, fully extended leg 0&deg;) knee extension torque. After a brief warm-up (5-min cycling, RPE 12-14), participants were seated and secured in the individually adjusted dynamometer. Participants were instructed to gradually increase their effort during three warm-up repetitions (50, 60 and, 70% of subjective maximal effort). After a 30-sec restitution period participants were instructed to perform three repetitions with maximal effort in the concentric phase. Sixty seconds after the isokinetic test the lever automatically moved to a 60&deg; angle and participants were instructed to push against the lever enough to see feedback from the visual feedback system. After an additional 15-sec restitution period, participants were instructed to push against the lever with maximal effort. Within the same assessment session, participants remained seated in the dynamometer for measurement perfomed on both legs. The first measurement was alternated between legs every other session. For statistical treatment of the data, all successful attempts were used. The last strength assessment at baseline was performed at least seven days prior to the first biopsy sampling. At least one of the baseline strength tests was performed on separate day with two sessions allowed to be perform on the same day with a short rest between assessments. Post training assessments were performed 48 hours and eight days after the last session.

For determination of body composition participants were scanned using DXA (Lunar Prodigy densitometer, GE Healthcare, Madison, WI, USA) with the standard scanning mode (13-25 cm). Participants were lying supine within the scanning bed reference lines, with a strap secured around the ankles to ensure a standardized body position in each scan. The scans were conducted with participants in a fasted state between 07.00-10.00 AM, with empty bladder and wearing only under-wear. Prior to each scan, a phantom scan was run to prevent baseline drifting from affecting analyses. The same technician was used at each time point. Analyses was performed using GE enCORE version 17.0 software (GE Healthcare). Region of interest was customized for covering upper thigh, marked with a sqaure from pubic symphysis to lateral part of tuberculum major, and distal to art. genu.

Muscle thickness (MT) was measured using a B-mode ultra sound unit (SmartUS EXT-1M, Telemed, Vilnius, Lithuania). Participants lay supine in a relaxed position for 20 min before assessments, with their feet strapped in a standardized position. A mark was set on the line 60% of the distance between Spinia Iliac Anterior Superior and the lateral femur condyle. MT of *m.vastus lateralis* was measured applying a water-soluble transmission gel (Aquasonic 100 Ultrasound Transmission Gel; Parker Laboratories Inc., Fairfield, NJ, USA), and a 39 mm 12 MHz ultrasound probe was placed perpendicular to the site of interest without pressing the skin. When the quality of the image was satisfactory, evident as distinct upper and lower muscle fascia, three images were captured, where the probe was relocated to the same position between each image. Position of the probe was marked on the skin and subsequently marked on a transparent paper to ensure similar probe placement for both the right and left *m.vastus lateralis* at subsequent assessments. Analyses were done in ImageJ Fiji [@RN2561] with images cropped and coded to ensure blinding of the assessor.

## Muscle biopsy sampling 
Muscle specimens were sampled bilaterally from *m. vastus laterlis* under local anesthesia (Lidokain 10 mg ml<sup>-1</sup>, Mylan, Mylan Ireland Limited, Dublin, Ireland) using a disposable needle (12-14 gauge, Universal plus, Medax, Poggio Rusco, Italy), operated with a spring loaded device (Bard Magnum, Bard Norway, Rud, Norway). Two to four passes were made to get sufficient material. One to two aliquots of the samples were quickly dissected free from connective and fat tissue, weighed and frozen in isopentan chilled to -80&deg;C and stored at -80&deg;C until further processing.<!-- At baseline, after twelve weeks of training and, after the de-training period, a further aliquot from each sample was selected for immunohistochemistry analysis and subsequently immersed in 4% formalin for fixation.-->        

## RNA and protein extraction
Frozen muscle tissue was homogenized in 1 ml of Trizol (ThermoFisher Scientific, Oslo, Norway) spiked with 0.04 ng of an external, non-mammalian, RNA spike-in (Lambda PolyA External Standard Kit, Takara Bio Europe, Saint-Germain-en-Laye, France). Mechanical disruption of the samples was achieved using Zirconium Oxide Beads (0.5 mm, Next Advance, Inc., New York, USA) and a bead mill (Bullet blender, Next Advance). Chloroform (200 &#956;l) was added prior to centrifugation (12000 g, 15 min at 4&deg;C) to achieve phase separation. Four hundred fifty &#956;l of the upper aqueous phase was transferred to a fresh tube and 500 &#956;l of isopropanol was added to precipitate the RNA. After a 10 min incubation at room temperature, samples were centrifuged (12000 g, 10 min at 4&deg;C), after which a pellet formed. The pellet was washed three times in chilled 75% ethanol with centrifugation between each wash (7500 g, 5 min at 4&deg;C). After the final wash all ethanol was removed and the pellet was eluted in 0.1X Tris-EDTA buffer. RNA concentration and purity was assessed by spectrophotometry. All samples had 260/280 ratios > 1.95.

Protein was extracted from Trizol preparations according to manufacturers instructions and [@RN2050]<!-- Kopec trizol protocol --> with modifications. The remaining aqueous phase was removed and DNA was precipitated by the addition of 300 &#956;l of absolute ethanol followed by gentle centrifugation (2000 g, 5 min at room temperature). An aliquot of the phenol-ethanol phase, corresponding to ~1.75 mg of tissue, was transferred to to a fresh tube. After addition of at least two volumes of isopropanol and incubation (10 min at room temperature), samples were centrifuged (7500 g, 10 min 4&deg;C) and a pellet formed. The pellet was washed three times in 95% ethanol with each wash separated by centrifugation (5000g, 5 min at room temperature). After the last wash all liquid was removed and 45 &#956;l of Kopec buffer [@RN2050] was added (5% SDS, 10 mM Tris, 140 mM NaCl and 20 mM EDTA, pH 8; containing protease and phosphatase inhibitors). Pellets were incubated at 50&deg;C for three hours after which the majority of samples were dissolved. Any undissolved material was sedimented by centrifugation (10000 g, 10 min at room temperature). Protein concentrations were measured (Pierce Detergent Compatible Bradford Assay, ThermoFisher Scientific). Sample were normalized in 4X Laemmli buffer (Bio-Rad Norway AS, Oslo, Norway), boiled (95&deg;C, 5 min) and stored at -20&deg;C before later use.

## Quantitative polymerase chain reaction (qPCR)
Complementary DNA (cDNA) was synthesized in technical duplicates from 500 ng of total RNA using random hexemer and anchored Oligo-dT primers (Thermo Fisher Scientific) together with Superscript IV (Thermo Fisher Scientific) according to manufacturer's instruction. qPCR reactions were performed with diluted cDNA (2 &#956;l, 1:25 dilution), a SYBR-green based commercial master mix (PowerUp™ SYBR™ Green Master Mix, Thermo Fisher) and, target-specific primers (500 nM) in 10 &#956;l reaction volumes using a real-time detection system (QuantStudio 5 Real-Time PCR System, Thermo Fisher Scientific). Fast cycling was used (1 sec denaturing, 30 sec annealing) after UNG (2 min, 50&deg;C) and polymerase  (2 min, 95&deg;C) activation. Melt curves were collected from all reactions to confirm single product amplification. Primers were further evaluated by agarose gel electrophoresis which confirmed primer sizes and non-template control experiments confirming no amplification without template. Primer sequences and their respective average performances are shown in Table 2.
```{r check-primer-sizes, include = FALSE}

## To check primer sizes ##
# Be aware that the gel is skewed (overestimating sizes in last wells)

#
# mw <- read_excel("./data/tr010_primers.xlsx", sheet = "molecular_marker") %>%
#  mutate(logx = log(X_coord)) %>%
#   print()
# mw_primers <- read_excel("./data/tr010_primers.xlsx", sheet = "primers_mw") %>%
#     mutate(logx = log(X_coord)) %>%
#   print()
# 
# 
# m1 <- lm(MW ~ poly(logx,3), data = mw)
#
# mw %>%
#  ggplot(aes(logx, MW)) + 
#  geom_point() + 
#  geom_smooth(method = "lm", formula = y ~ poly(x, 3))
# 
# 
# mw_primers$mw <-predict(m1, newdata = mw_primers)
# 
# 
# library(ggrepel)
# read_excel("./data/tr010_primers.xlsx") %>%
#   mutate(predicted_product_size = rstart - fstart) %>%
#   dplyr::select(primer = primer_id, predicted_product_size) %>%
#   inner_join(mw_primers) %>%
#   ggplot(aes(predicted_product_size, mw, label = primer)) + 
#   geom_point() + 
#   ggrepel::geom_text_repel() + 
#   scale_x_continuous(limits = c(50, 160)) +
#   scale_y_continuous(limits = c(50, 160))+
#   geom_abline(slope = 1, intercept = 0)
# 



```
Raw fluorescence data was exported from the QuantStudio software and estimates of quantification cycle (Cq) and amplification efficiency was derived for each reaction using the qpcR package[@RN1768].

## Immunoblotting
Protein samples (20 &#956;g) were separated on 4-20% Tris-Glycin gels (Criterion TGX Precast Gels, Bio-Rad) at 250 V for 45 min using the recommended running buffer (25 mM Tris, 192 mM Glycin, 0.1% SDS). All samples from the same participant were run on the same gel and all samples were run in at least duplicates. Separated samples were transferred to &#956m PVDF membranes (Immun-Blot, Bio-Rad) using wet transfer (25 mM Tris, 192 Glycin, 10% vol/vol methanol) at a constant voltage of 300 mA for 3 h. Membranes were then stained to confirm transfer and enable total protein quantification using a reversible protein stain (Pierce Reversible Protein Stain, Thermo Fisher Scientiﬁc). Primary antibodies were acquired to detect UBF (F-9, sc-13125, Santa-Cruz Biotechnology, Dallas, Texas, USA) and ribosomal protein S6 (54D2, #2317, Cell Signaling Technology, Danvers, MA, USA). After blocking (Tris-buffered saline blocking buffer, 20 mM Tris, 150 mM NaCl, 5% fat-free milk, 0.1% Tween-20), membranes were incubated over-night with primary antibodies diluted in blocking buffer (UBF, 1:200; S6, 1:1000) followed by incubation with a secondary antibody conjugated to horseradish peroxidase (Anti-mouse IgG, #7076, Cell Signaling Technology, 1:10000). Membranes were washed 6 &#215; 5 min after incubation with primary antibodies and 8 &#215; 5 min after incubation with the secondary antibody. All incubation and washing steps were performed at 4&deg;C using an automatic membrane processor (BlotCycler, Precision Biosystems, Mansﬁeld, MA, USA). Chemiluminescent signals from membranes were detected after 5 min incubation in substrate (Super Signal West Femto Maximum Sensitivity Substrate, Thermo Fisher Scientiﬁc) using a documentation system. Total protein content was quantified from whole membrane images and defined as the mean gray value of the whole lane. Between-lane gray values were used as background subtracted from protein values. Total protein quantification was done using ImageJ Fiji [@RN2561]. Chemiluminescence signals were quantified using Image Studio Lite (LI-COR Biotechnology, Lincoln, NE, USA).

## Statistics and data analysis
Descriptive data are presented as mean and standard deviation (SD). The effect of training on muscle strength, muscle thickness, UBF/rpS6 protein, total RNA and gene abundances were assessed using mixed effects regression models. Time and group (TRAIN vs. CTRL) were treated as population (fixed) effects and leg nested within participant included as group level (random) effects. These analyses were performed on data with matching time points between TRAIN and CTRL with the exeption that all post-training data from TRAIN were included (post-training and de-training). Relative interactions between groups were estimated as $\Delta$ TRAIN - $\Delta$ CTRL. The effects of different volume conditions and general time-course patterns were assessed using all pairwise observations from the TRAIN group. For protein and total RNA data, segmented regression models were used to estimate changes over sessions in three segments (session 1-4, 4-8 and 8-12; corresponding to blocks of different volume prescription in TRAIN). When no robust effects of volume conditions were detected, group averages are presented. Segmented models were fitted with time and volume condition as population effects and legs nested within participants as group level effects.

Muscle strength, muscle thickness, protein and total RNA data was modeled after log transformation.
Gene abundance data were fitted with number of sessions as a categorical variable in comparisons of volume conditions, and Cq values converted to counts as suggested by Matz et al. [-@RN1964]. A Poisson-lognormal model was used to fit these count, using data from all genes and including group level effects for each technical duplicate, controlling for technical errors during sample preparation. An offset consisting of a normalization factor proportional to the amount of muscle used to prepare cDNA was used to model gene abundance per tissue weight. The external reference gene was used to calculate the normalization factor (External reference counts $\times$ muscle weight (mg) in each Trizol preparation). The offset was specified as a predictor with the coefficient fixed to 1.

A linear model was used to estimate the increase per session and average total RNA for every leg in the TR group. These estimates were then used to estimate the effect of training-induced increase per session and average total RNA abundance on muscle hypertrophy. For each leg, session was used as the independent variable centered on Session 6 and log transformed RNA per tissue weight as the dependent variable. Mean-centering of the independent variable was done to obtain an estimate of the average RNA concentration per leg. This also assured that the slope and intercept did not correlate, something that could lead to colinearity issues in subsequent modeling. A mixed effects model was subsequently fitted with differences in muscle thickness pre- to post-training as the dependent variable and estimated percentage per session increases in total RNA, the mean total RNA scaled as standard deviations from the mean and sex as independent variables. Leg nested within participant was used as group levels effects. The robustness of this model was assessed by leave-one-out analysis on the level of individual data points in the relationship between total RNA and sessions and on the level of participants (see Results).

All models were fitted using a Bayesian framework using either the brms [@RN2562] or MCMCglmm [@RN1992] package written for R [@Rcore]. Inference about effects of interest was done based on point estimates and their 95% credible intervals (CI). Credible intervals not containing null effects were interpreted as robust. Models were fitted with default priors which also makes CI analogous to confidence intervals but with the interpretation being that the CI contains the true population value with the specified certainty (95%), given the data.
Fitting performance was assessed by confirming convergence of at least four different chains of MCMC samples (graphically assessed and confirmed with $\hat{R}\approx 1$). Model performance was assessed from comparing simulated data from each model to observed data graphically (posterior predictive checks). 


# Results
```{r training-data}

m2 <- readRDS("./data/derivedData/training-data/m2.RDS")
# Model containing interaction 
# readRDS("./data/derivedData/training-data/m1.RDS")


# Create a hypothesis and convert to percentages


h <- hypothesis(m2, c("(weekW2 / Intercept) * 100 = 0",
                 "(weekW3 / Intercept) * 100 = 0"))$hypothesis 



load.inr.w2 <- paste0(sprintf("%.0f",h[1, 2]), 
                      "%, 95% CI: [",
                      sprintf("%.0f",h[1, 4]), 
                      ", ",
                      sprintf("%.0f",h[1, 5]), 
                      "]")
load.inr.w3 <- paste0(sprintf("%.0f",h[2, 2]), 
                      "%, [",
                      sprintf("%.0f",h[2, 4]), 
                      ", ",
                      sprintf("%.0f",h[2, 5]), 
                      "]")


```

In TRAIN, all participants successfully completed their prescribed RT on both legs, with the two volume conditions resulting in diverging volume profiles (load &#215; repetitions) over the course of the study (Figure 1B). Exercise intensities (resistance at 10RM) increased similarly between conditions from the first to the second (`r load.inr.w2`) and third (`r load.inr.w3`) training block, with each block consisting of four training sessions. Isokinetic strength increased from baseline to after Session 12 in TRAIN compared to CTRL. This difference was still seen after de-training (Figure 1C). Isometric strength showed the same general pattern, though with considerably larger degrees of uncertainty, as indicated by  wider 95% CI (Figure 1C). Muscle thickness of *m. vastus lateralis* increased in TRAIN compared CTRL, measured both after Session 12 and after eight days of de-training (Figure 1D). There were no indications of differences between volume conditions in strength or muscle thickness changes.

```{r western-data}
segmented_protein_results <- readRDS("./data/derivedData/western-analysis/segmented_results.RDS")



s6_slope0_4 <- paste0(sprintf("%.1f", 100 * (exp(segmented_protein_results[6,4]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[6,6]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[6,7]) - 1)),
                   "]")
                      
s6_slope4_8 <- paste0(sprintf("%.1f", 100 * (exp(segmented_protein_results[7,4]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[7,6]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[7,7]) - 1)),
                   "]")
                      

s6_slope8_12 <- paste0(sprintf("%.1f", 100 * (exp(segmented_protein_results[8,4]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[8,6]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[8,7]) - 1)),
                   "]")

ubf_slope0_4 <- paste0(sprintf("%.1f", 100 * (exp(segmented_protein_results[14,4]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[14,6]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[14,7]) - 1)),
                   "]")
                      
ubf_slope4_8 <- paste0(sprintf("%.1f", 100 * (exp(segmented_protein_results[15,4]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[15,6]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[15,7]) - 1)),
                   "]")
                      

ubf_slope8_12 <- paste0(sprintf("%.1f", 100 * (exp(segmented_protein_results[16,4]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[16,6]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[16,7]) - 1)),
                   "]")
                      

     

s6.m3 <- readRDS("./data/derivedData/western-analysis/s6_detrain.RDS")
ubf.m3 <- readRDS("./data/derivedData/western-analysis/ubf_detrain.RDS")



ubf.sum <- summary(ubf.m3)$fixed


var_s12 <- paste0(sprintf("%.1f", 100 * (exp(ubf.sum[3,1]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(ubf.sum[3,3]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(ubf.sum[3,4]) - 1)),
                   "]")

const_detrain <- paste0(sprintf("%.1f", 100 * (exp(ubf.sum[2,1]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(ubf.sum[2,3]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(ubf.sum[2,4]) - 1)),
                   "]")

var_const_interaction <- paste0(sprintf("%.1f", 100 * (exp(ubf.sum[4,1]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(ubf.sum[4,3]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(ubf.sum[4,4]) - 1)),
                   "]")

h <- hypothesis(ubf.m3, "condvar + timepost1w:condvar = 0")$hypothesis %>%
  data.frame()

var_change <- paste0(sprintf("%.1f", 100 * (exp(h[1,2]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(h[1,4]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(h[1,5]) - 1)),
                   "]")


```

UBF and rpS6 protein levels were increased only after twelve sessions in the training group compared to control and stayed elevated after eight days of rest (Figure 2A and B), with no changes being observed after the first training session (48 hrs). For both rpS6 and UBF, protein levels increased linearly throughout the training intervention, with rpS6 showing estimated increments per session corresponding to `r s6_slope0_4` during block 1 (session 1-4), `r s6_slope4_8` during block 2 (session 5-8) and `r s6_slope8_12` during block 3 (session 9-12) and UBF showing increments corresponding to `r ubf_slope0_4`, `r ubf_slope4_8` and `r ubf_slope8_12`. Slopes did not differ between volume-conditions but there was a tendency towards lower levels of UBF in VAR after session 12 (`r var_s12`). After the de-training period CONST tended to show decreased levels of UBF (`r const_detrain`) while VAR remained at elevated level compare to after session 12 (`r var_change`; interaction effect: `r var_const_interaction`). Levels of rpS6 did not change in response to the de-training period and remained similar between volume conditions. UBF was robustly up-regulated at the mRNA level compared to CTRL after 48 hrs (Figure 2D) with no other robust differences seen between TRAIN and CTRL or between volume conditions in TRAIN in either UBF or rpS6 mRNA levels (Figure 2D and E). 


```{r total-rna}

# Segmented model analysis in text 
# Read the segmented model 
tc.m2 <- readRDS("./data/derivedData/total-rna-analysis/tc.m2.RDS")




time1.hyp <- hypothesis(tc.m2, "time1 = 0")
time2.hyp <- hypothesis(tc.m2, c("time2 + time1 = 0")) # The slope (on log scale) between session 4 and 8
time3.hyp <- hypothesis(tc.m2, c("time2 + time1 +time3 = 0")) # The slope on log scale between session 8 and 12

slope0_4 <- paste0(sprintf("%.1f", 100 * (exp(time1.hyp$hypothesis[1,2]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(time1.hyp$hypothesis[1,4]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(time1.hyp$hypothesis[1,5]) - 1)),
                   "]")
                      
slope4_8 <- paste0(sprintf("%.1f", 100 * (exp(time2.hyp$hypothesis[1,2]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(time2.hyp$hypothesis[1,4]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(time2.hyp$hypothesis[1,5]) - 1)),
                   "]")
                      

slope8_12 <- paste0(sprintf("%.1f", 100 * (exp(time3.hyp$hypothesis[1,2]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(time3.hyp$hypothesis[1,4]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(time3.hyp$hypothesis[1,5]) - 1)),
                   "]")

# Estimate changes to end of periods

sessions <- hypothesis(tc.m2, c("time1 * 4 = 0", 
                                     "(time1 * 8) + (time2 * 4)= 0", 
                                     "(time1 * 12) + (time2 * 8) + (time3 * 4)= 0"))



                  

## Effect of detraining 

detrain.hyp <- hypothesis(tc.m2, "detraindetrain = 0")

detrain_rna <- paste0(sprintf("%.1f", 100 * (exp(detrain.hyp$hypothesis[1,2]) - 1)),
                           "%, [",
                      sprintf("%.1f", 100 * (exp(detrain.hyp$hypothesis[1,4]) - 1)), 
                   ", ",
                       sprintf("%.1f", 100 * (exp(detrain.hyp$hypothesis[1,5]) - 1)),
                   "]")
                      
fourth.session <- paste0(sprintf("%.1f", 100 * (exp(sessions$hypothesis[1,2]) - 1)),
                           "% [",
                      sprintf("%.1f", 100 * (exp(sessions$hypothesis[1,4]) - 1)), 
                   ", ",
                       sprintf("%.1f", 100 * (exp(sessions$hypothesis[1,5]) - 1)),
                   "]")


eight.session <- paste0(sprintf("%.1f", 100 * (exp(sessions$hypothesis[2,2]) - 1)),
                           "% [",
                      sprintf("%.1f", 100 * (exp(sessions$hypothesis[2,4]) - 1)), 
                   ", ",
                       sprintf("%.1f", 100 * (exp(sessions$hypothesis[2,5]) - 1)),
                   "]")

twelth.session <- paste0(sprintf("%.1f", 100 * (exp(sessions$hypothesis[3,2]) - 1)),
                           "% [",
                      sprintf("%.1f", 100 * (exp(sessions$hypothesis[3,4]) - 1)), 
                   ", ",
                       sprintf("%.1f", 100 * (exp(sessions$hypothesis[3,5]) - 1)),
                   "]")

```

A single session of RT led to robust increases in pre-rRNA 47S and 45S abundance, measured as expression per unit tissue weight, both from baseline to 48 hrs after session 1 within TRAIN (Figure 3B) and compared to CTRL (Figure 3C).
Similarly, 47S and 45S abundances were also increased after twelve sessions, together with 45S ITS and rRNA 28S and 18S species, measured both within-participant and compared to CTRL (Figure 3C).
After eight days of rest, only 18S and 28S remained at elevated levels compared to CTRL (Figure 3B).
This general pattern of rRNA expression was reflected by total RNA expression per unit tissue weight, which were robustly increase in TRAIN compared to CTRL after session twelve (Figure 3E and F), followed by decreased levels after de-training `r detrain_rna`.
For both rRNA expression and total RNA levels, these training-associated increases in abundance predominately occured during the first four sessions, evident as `r slope0_4` increase per session, followed by sustained levels from sessions four to eight `r slope4_8` and from sessions eight to twelve `r slope8_12`. This corresponded to an increase from baseline by `r fourth.session`, `r eight.session` and `r twelth.session` to 48 hrs after session four, eight and twelve, respectively.
In TRAIN, the two volume conditions were associated with minor differences in rRNA subspecies abundance, with only 45S ETS showing differential expression between conditions, evident as robustly higher levels in VAR compared to CONST after the 12th session (Figure 3D), coinciding with increased volume in this condition towards the end of the intervention.
No robust differences were seen between conditions for total RNA abundances (Figure 3G), increasing similarly in both conditions.

```{r combined-analysis}

# Save model 
comb.m1 <- summary(readRDS("./data/derivedData/ubf-tot-rna-model/comb_m1.RDS"))$fixed




ubf_cor_tot_rna <- paste0(sprintf("%.1f", 100 * (exp(comb.m1[2,1]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(comb.m1[2,3]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(comb.m1[2,4]) - 1)),
                   "]")



```

In TRAIN (after for controlling for time), UBF levels robustly predicted total RNA levels with a one unit difference in UBF levels corresponding to one standard deviation leading to `r ubf_cor_tot_rna` increases in total RNA per unit tissue weight (Table 3). In contrast, no evidence for a relationship between total RNA and rpS6 was found when controlling for time (Table 3). 
To estimate the influence of total RNA on changes in muscle mass in TRAIN, measured as *m.vastus lateralis* thickness, the average linear increase in total RNA was calculated for each leg together with the average total RNA abundance over the course of the training intervention.<!-- (estimated as the intercept in a regression model with number of sessions centered at session 6). move to methods -->
The relationship between muscle growth and characteristics of individual total RNA profiles were estimated using a regression model containing the increase in total RNA concentrations, expressed as percentage increase per session, and average total RNA abundance at Session 6, expressed as standard deviations from the mean.
The model also contained sex as predictor to account for sex differences in muscle growth.
Total RNA increase in response to training RNA was robustly related to muscle growth (Table 4, Figure 4A). Average total RNA levels tended to negatively influence muscle growth (Table 4, Figure 4B). To assess the robustness of the model, individual relationships between sessions and total RNA levels (Figure 4C) were recalculated after removal of single data points. The model was refitted using new estimates of the total RNA abundance and increase per session. Effects of refitting the model is shown in Figure 4D. This analysis indicated that the relationship between Total RNA increase and muscle growth was preserved in all iterations (Figure 4D). We further assessed the model by iteratively removing one participant from the data set, similarly this showed that estimates of the effect of total RNA increase on muscle growth was robust but the effect of average total RNA estimates were more variable (Participant 8 and 3 in Figure 4D).


# Discussion
Here, we confirm that resistance training leads to increased abundance of markers of ribosome density, measured as total RNA, ribosomal RNA subspecies and rpS6 protein in previously untrained individuals compared to a non-training control group. We show that this increase is interconnected with increases in UBF protein abundance, suggesting a plausible manner for rRNA transcription regulation in response to RT. We did not find any convincing support for the notion that ribosomal biogenesis is tightly regulated by changes in training volume, however, training cessation may lead to halted ribosome biogenesis. Finally, the rate of increase in total RNA abundance predicted the magnitude of muscle growth. 

Several studies have presented evidence of a determinant role for ribosomal biogenesis in RT induced muscle hypertrophy. Increases in total RNA has been shown to positively correlate with RT induced hypertrophy
[@RN1644;<!-- Figueriedo  -->
@RN1897;
@RN2055].<!-- Mobely Three groups, no clear group * time in RNA incrinteraction but 32 vs 8 % increase in high vs low responders, although correlation RNA incr with MT incr -->
<!-- Reidy correlation between total RNA and muscle hypertrophy-->
and differs between high- and low-responders to RT
[@RN1755].<!-- Stec 2016  -->
Additionally, volume-dependent regulation of total RNA predicted beneficial effects of high- vs. low-volume RT
[@RN2358]. In addition to observational evidence from human studies, blockage of ribosomal biogenesis halts muscle cellular growth *in vitro* in some
[@RN1632;<!-- Nader 2005 blockage of mTOR halts ribo bio and growth -->
@RN1754<!-- West 2016 Inhibition of c-Myc (through inibitor) reduces RNA and MPS-->;
@RN1755]<!-- Stec specific pol I inhibition -->
but not all situations
[@RN1929]<!-- Crossland 2017 inhibition of ribo bio did not limoit igf-1 stimulated growth  -->
Results from the present study corroborates the link between ribosomal biogenesis and muscle hypertrophy.
When using all available data points we were able to robustly estimate both the rate of increase in total RNA and the average RNA levels over the course of the study and include both when modeling changes in muscle thickness.
The results of this analysis showed that the rate of RNA accumulation determined muscle growth when average levels of total RNA was controlled for.
In other words, at a fixed relative rate of increase in total RNA, legs with higher levels of RNA did not grow more but at a fixed amount of total RNA, legs with higher increase per session did.
This suggest that the absolute ribosomal density is not as important as the rate of increase in ribosomal density.
In an homogeneous group, in response to RT, the rate of increase will be highly correlated with maximal values seen after RT, linking maximal rate to the resulting abundance of ribosomes.
The same pattern revealed in our model is seen when comparing young and old muscle, where aged muscle display higher levels of total RNA at rest
[@RN1656]<!-- Stec 2015 higher levels of RNA/mg in old -->
and lower levels of increase in response to RT
[@RN1809].<!-- Brook 2016 increase in RNA per mg not seen in old, data at baseline collected at different timepoints? -->
Together with our results this underlines the importance of understanding RT induced ribosomal synthesis in relation to manipulation of training loads for specific populations.

<!-- rpS6 dissconect from total RNA levels. -->
In the present study, training induced increases in rRNA and total RNA coincided with increases in rpS6. Changes in total RNA levels and rpS6 in response to de-training did however not correspond as rpS6 protein levels remained elevated after the de-training period. Training induced increases in rpS6 seen in the present study are in agreement to what has previously been reported in young men
[@RN1642],<!-- Brook 2015 rpS6 increase in resposne to RT over control -->
but not in elderly men and women where a decrease was observed in response to training despite increases in total RNA and rRNA
[@RN1755].<!-- Stec 2016 rpS6 decreased 22% after four weeks of training in older  -->
Although increases were seen in both rpS6 and and total RNA, rpS6 did not explain variations in total RNA when number of sessions were controlled for. Together with a disconnect after the de-training period, this may suggest that regulation of rpS6 expression and transcription of ribosomal RNA displays different temporal characteristics resulting from RT. Additionally, ribosomal proteins may have extra-ribosomal functions 
affecting their expression
[@RN2588].<!-- Warner 2009 General description of   -->  

Total RNA is the most studied proxy marker of ribosomal density as most of the RNA is assumed to be ribosomal [@RN2223].<!-- Young  -->
This marker has been shown to be sensitive to RT in numerous studies 
<!--@RN2266<  Hortobagyi 2000 NO change in RNA per wet weight after immobilization or 12 weeks retraining low N-->
[@RN2225;<!-- Haun 2019 used a progressive volume troughout a six week study study RNA increased during six weeks (12% week 3; 27% week 6 -->
@RN2180;<!-- Haddad 2005 Unloading through limb suspension leads to lowered total RNA levels, resistance training increased --> 
@RN1656;<!-- Stec2015 A single session ns ~ 9% increase in young, -4% in old -->
@RN1520;<!-- Bickel 2005 Two bouts electric stimulation, young ~ 30% incr-->
@RN2358; <!-- Four sessions ~29 and 18 % in mod vs. low volume|31 sessions 18/12% young-->
@RN1755<!-- Stec 2016 12 sessions 26, 9 and 6% increase in high, mod and low responders OLD-->;
@RN1644;<!-- Figuieredo 2015 16 sessions 49% increase young -->
@RN1809; <!-- Brook 18 sessions 8 and 40% (9 sessions) and 15,22% (18 sessions) old and young-->
@RN1897<!-- 11% 36 sessions young -->].
However, a detailed time course has not been mapped.
To date, available data suggests that maximal levels are reached within four to nine sessions in young males and females
[@RN1809;
@RN2358].
We did not detect any meaningful further increases after 48 hrs after the eight session were peack the peack average increase was 50% from baseline. This confirmed a plateau when total RNA was expressed per unit tissue weight. A possible reason for the plateau could be attenuated ribosomal biogenesis. However, pre-rRNA abundances per unit tissue weight remained at elevated levels indicating retained transcriptional activity after the twelfth session together with maximal levels of UBF protein. Another possibility is a dilution effect
[@RN2142]<!-- Figueiredo review details the concept of rna concentration -->
due to muscle hypertrophy and thus protein accretion.
After the twelfth session, muscle hypertrophy was apparent and did not change in response to the de-training period.
As muscle hypertrophy did not change from after session twelve to after the de-training period, the apparent decrease in total RNA was not driven by dilution but by attenuated rRNA transcription.
This notion is supported by normalization of pre-rRNA levels and possibly by lowered levels of UBF protein, although this apparent effect was not confirmed as statistically robust.

Over the full training period UBF levels did robustly explain total RNA levels after accounting for time, something that otherwise would lead to biased estimates as both the dependent variable (total RNA) and the covariate (UBF levels) varies with time.
From a mechanistic perspective, UBF is an important factor for rDNA transcription as it, in its active state recruits a secondary transcription factor (SL1) to the rDNA promoter and enables transcription by RNA polymerase I
[@RN2563].<!--Lin 2007 phospho of UBF shows it has a role in recruioting SL1 to the rDNA promoter -->
Activation of UBF is controlled by signaling from the mechanosensitive mTOR, pathway as rapamycin, a specific mTOR inhibitor, blocks UBF from recruiting SL1 and subsequent rRNA transcription
[@RN1632; <!-- Nader rapamycin block UBF availability  -->
@RN2564]<!-- Hannan 2003 -->
<!-- Hannan 2003 10.1128/MCB.23.23.8862-8877.2003 mTOR is believed to play a role in phosphorylating UBF leading to recruitment of SL-1. The effect of blockage of mTOR through rapamycin is not rescued by hypophosphorylated UBF -->
Evidence from human exercise studies confirms training-induced activation of UBF through phosphorylation
[@RN1746; <!-- Figueiredo 2016 cold water inhibits UBF activation and increase rRNA increase -->
@RN1644].<!-- Figueriedo ubf increase total and active form (ser388) --> 
In addition to exercise-induced activation of UBF, mechanical loading also leads to increased levels of total UBF
[@RN1746; <!-- Figueiredo 2016 cold water inhibits UBF activation and increase rRNA increase -->
@RN1644]. Increases in UBF was determined to be rapamycin insensitive after synergist ablation in mice
[@RN1072]<!-- Goodman 2011 UBF increase due to synergist ablation is rapamycin insensitive https://doi.org/10.1113/jphysiol.2011.218255 --> 
pointing to an effect observed in cell models where c-Myc induces UBF mRNA transcription [@RN2565].<!-- Pootinga 2004 MAD1 and c-Myc controls UBF transcription UBF is required for c-Myc induced rDNA transcription -->
Interestingly the avalability of UBF *per se* has been shown to regulate rRNA transcription
[@RN2566]<!-- Hannan 1996 over expression of UBF leads to rRNA transcription in cardiomyocytes -->  through control of rDNA gene activity
[@RN2556].<!-- Sanij, UBF depleting leads to inactivation of rDNA genes -->
Together with our observations, this underlines the importance of UBF as a regulator of RT-induced ribosomal biogenesis.

Resistance training volume is known to be a potent modulator of molecular mechanisms determining protein synthesis and ribosomal biogenesis including c-Myc induction, mTOR activation
[@RN2358; <!-- Hammarström volume affects ribo biogenesis and mTOR and c-myc -->
@RN1837; <!-- Ahtianinen 2015 volume affects mTOR differently -->
@RN791]<!-- Burd ex volume protein synthesis and anabolic [rpS6] phospho signalling-->,
subsequent total RNA increases [@RN2358] and post exercise protein synthesis [@RN791]
and subsequent training outcomes [@RN1767; @RN2358].<!-- Schoenfeld 2016 meta and Hammarström -->
We have previously shown that different amounts of training volume in constant volume protocols leads to differences in total RNA and rRNA accumulation in the initial phase of RT [@RN2358].
Based on the notion that ribosomal biogenesis is a complex and energy demanding process
[@RN1920],<!-- Warner economics of ribosomal biogenesis -->
we hypnotized that fluctuations in training load would reflect in markers of ribosomal biogenesis.
When comparing the two volume conditions in the present study we only found one part of the 45S ETS to be differentially expressed at Session 12 in favor of VAR together with a tendency of a rescued UBF level after de-training in response to increased volume in the VAR protocol.
These observations do not give much support for a clear effect of fluctuations in training volume in the short time span used in the present study although the time point with increased 45S ETS expression was preceded by a period of increased training volume, suggesting a potential interaction between time and volume.
Indeed, both training protocols utilized in the present study also increased strength and induced muscle hypertrophy to a similar degree.
From a general perspective, although volume is an important factor for increases in muscle strength and mass
[@RN1767;<!-- schoenfeld 2016 dose response  -->
@RN2492]<!-- Ralston 2017 Weekly sets volume on muscle strength  -->
different organization of training loads over time is likely of minor importance when training volumes are equated
[@RN2572].<!-- Grgic 2017 volume equated periodization schemes led to similar muscle hyeprtrophy -->
It is important to note that RT in the current study was performed with constant volume in the first four sessions, something that could have been more than enough to maximize rRNA transcription in previously untrained individuals.
This is supported by the observation that pre-rRNA increased rapidly initially in both protocols with minimal changes in response to subsequent sessions, regardless of exercise volume.
The CONST protocol in the present study corresponded to volumes used in the high volume condition in a previous study from our lab (three sets in two exercises activating knee extensor muscles) [@RN2358].
There, higher levels of total RNA accumulation were observed compared to a low volume protocol [@RN2358].
Interestingly, using a progressive volume protocol in well-trained participants, increases in total RNA have been reported throughout six weeks of training
[@RN2055].<!-- Mobley 2018 trend towards higher increases in high responders 32% high resp, 8% low resp 12 wk training -->
Compared to the present study and other using constant volume protocols
[@RN1809;<!-- Brook 2016 -->
@RN2358],<!-- Hammarström 2020 -->
progressive volume may thus increase ribosomal abundance to a higher degree and provide a measure to avoid the plateau phase seen in the present study.
This observation was done in well-trained participants performing a high volume protocol without a control group condition with constant volume.

In conclusion, RT induced ribosome accumulation reaches peak values in the initial phase of RT (8-12 sessions) affected by total levels of UBF. Fluctuations in training volume does not transfer to fluctuations in ribosomal biogenesis a moderate volume initiation of RT but training cessation does lead to attenuated ribosomal biogenesis. The rate of total RNA accumulation predicts RT induced muscle hypertrophy.  


<!-- NOTES -->
<!-- UBF and S6 protein levels may be differently regulated in men and rats as more consecutive bouts of electrical stimulation in rats does not prtoduce the same pattern seen in our study 
 Kotani 2019 10.1152/japplphysiol.00665.2018 increase in t UBF at 6 h post ex but not 48 despite 2 weeks training (3 x week). S6 increase at 48 h and 6 bouts but return to sed values at 18 bouts. 18 and 28 S increased over sed at 48 h post ex with no further increase beyond 3 bouts 45S maximal at three bouts -->







# References
<div id="refs"></div>

# Figure legends

**Figure 1.** Study design (A) training load (B) and training outcomes (muscle strength, C; muscle thickness D).   

**Figure 2.** Protein (A and B) and mRNA abundances (D and E) of rpS6 and UBF. Black points and error bars represents statistically robust results (a 95% CI not containing 0). C shows western-blots and total protein stains from a representative participant. mRNA data is normalized per total RNA.  

**Figure 3.**  Total RNA and ribosomal RNA subspecies in response to trainin. Total RNA increased compared to non-training controls to post-training (12 sessions) and tended to normalize after de-training (A). There were no robust differences between different volume conditions in the training group (A). B shows primer locations targeting different ribosomal RNA subspecies. Subsets of ribosomal RNA showed robust increases compared to control 48 h after the first session and other after the training period (C). Error bars shows 95% CI. Asterisk in C indicates robust differences between volume conditions (a 95% CI of pairwise differences not containing 0).  

**Figure 4.** 
Predictions of muscle thickness increase based on total RNA increases (A) and total RNA abundance (B; see Table 4). Values are averaged over values from men and women. Individual plots of estimates total RNA increases over time is shown in C together with results from leave-one-out analysis (D). Leave-one-out analysis shows the effect of removing a single participant (black point and error-bars) and individual values from the total RNA per time estimates where green points represents bounds of the 95% CI and yellow points represents mean estimates.  



  
