---
title: "Ribosome accumulation during early phase resistance training"
author: Daniel Hammarström^1,2,£^&ast;,Sjur J. Øfsteng^2,£^, Nicolai B. Jacobsen^1^, Krister B. Flobergseter^1^,  Bent R. Rønnestad^1^, Stian Ellefsen^1,3^
date: ""
header-includes:
  - \usepackage{wasysym}
csl: resources/experimental-physiology.csl
bibliography: resources/bibliography.bib
output:
  word_document:
    reference_docx: resources/manuscript_word_template.docx
editor_options: 
  chunk_output_type: console
---
<!-- This is a comment  -->
<!-- Add comments after each reference for tracking (eg[@RN2364]<!-- comment -->
<!-- After references, add a row break without space (this will put the reference on the end on the line). -->

<!-- The formatting of the manuscript is based on faseb journal, find csl-files at https://github.com/citation-style-language/styles -->

<!-- Figure texts are included in the end of the manuscript file -->
<!-- Tables in a separate file -->

```{r, include = FALSE}

source("./R/libs.R")
# Settings for chunk options
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)

```


^1^ Section for Health and Exercise Physiology, Department of Public Health and Sport Sciences, Inland Norway University of Applied Sciences, Lillehammer, Norway. 

^2^ Swedish School of Sport and Health Sciences, Stockholm, Sweden.

^3^ Innlandet Hospital Trust, Lillehammer, Norway. </div>

^£^ These authors contributed equally to this work




<div custom-style="correspondence_style">
&ast; Correspondance

Daniel Hammarström, Section for Health and Exercise Physiology, Department of Public Health and Sport Sciences, Inland Norway University of Applied Sciences, Lillehammer, Norway. Email: daniel.hammarstrom@inn.no  </div>



<div custom-style="runningtitle_style">**Running title:** Ribosome biogenesis in early phase resistace training</div>


\pagebreak


# Abstract
<div custom-style="abstract_style">
<!-- Word limit 200 -->
Abstract...


**Keywords:** Resistance training, ribosome biogenesis


</div>

\pagebreak

# Introduction

Skeletal muscles not only constitutes our locomotive apparatus but also serve as our primary storage site of amino acids. This makes the skeletal muscle system a critical target for interventions that promotes health across the lifespan [@RN763]<!-- Wolfe 2006 -->. 
Resistance training can promote beneficial effects on skeletal muscles and does so through exercise-induced changes in the balance between muscular protein breakdown and synthesis, with successful RT leading to net protein accretion and muscle hypertrophy. A traditional view of exercise-induced muscle hypertrophy has been that muscle protein accumulates as a consequence of repeated episodes of elevated post-exercise protein synthesis in the acute phase, resulting in a net-positive protein balance over time. Indeed, one bout of resistance exercise acutely up-regulates skeletal muscle protein synthesis up to 48 hr after exercise [@RN786]<!-- Phillips 1997 -->
and subsequently, repeated bouts would lead to accumulation of muscle protein over time
[@RN1272<!-- Phillips review 2014 -->;
@RN1642]<!-- Brook 2015 experiment -->. 
In recent years, this view has been supplemented with evidence showing that chronic RT also leads to an increased resting synthetic rate of muscle protein
[@RN1521;<!-- Kim 2005 Fasted state skeletal muscle PS altered with training  --> 
@RN1866;<!-- Wilkingson 2008 -->
@RN1897]<!-- Reidy 2017 Increased rested state protein synthesis -->.
An increase in the resting protein synthetic rate has been postulated to be related to an increased translational capacity, i.e. accumulation of ribosomes
[@RN1897;<!-- Reidy 2017 -->
@RN1912]<!-- Figueiredo 2017 Ribosome biogenesis review -->,
a notion supported by observations relating exercise-induced changes in total RNA, a proxy marker for ribosome abundance, to muscle hypertrophy
[@RN1644;<!-- Figueiredo 2015 Correlation between total RNA fold change and CSA hyp -->
@RN1755; <!-- Stec Ribosome bio higher in high responders -->
@RN2358]<!-- Volume dependent RNA accum relates to volume depend hyper -->.
Furthermore, inhibition of ribosomal RNA (rRNA) transcription or inhibition of up-stream transcription factors leads to halted biosynthesis of ribosomes and diminished cellular growth
[@RN1755<!-- Stec 2016 Pol I-specific inhibition leads to GF induced myotube hypertrophty -->;
@RN1754<!-- West 2016 Inhibition of c-Myc (through inibitor) reduces RNA and MPS-->;
@RN1810]<!-- von Walden 2016 Inhibition of Pol 1 rDNA tracsription and myotube growth -->.  

Formation of new ribosomes is a complex and energy demanding process involving the whole transcriptional apparatus
[@RN1920;<!-- Warner review yeast -->,
@RN1820;<!-- Moss review -->
@RN1810]<!-- Inhibition of Pol II affects trasncription at Pol I -->.
In skeletal muscle, a single bout of resistance exercise elevates markers of ribosome biogenesis in skeletal muscle evident from studies investigating immature forms of rRNA in response to exercise in the untrained state
[@RN1656<!-- A single session upregulates 45 S 24 h post ex -->;
@RN1037<!-- Nader 2014 45S up in response to 4 h after acute exercise -->].
Studies examining repeated bouts of resistance exercise suggests that RT leads to accumulation of ribosomes, measured as total RNA, with increased number of completed sessions
[@RN1656;<!-- Stec2015 A single session ns ~ 9% increase in young, -4% in old -->
@RN1520;<!-- Bickel 2005 Two bouts electric stimulation, young ~ 30% incr-->
@RN2358; <!-- Four sessions ~29 and 18 % in mod vs. low volume|31 sessions 18/12% young-->
@RN1755<!-- Stec 2016 12 sessions 26, 9 and 6% increase in high, mod and low responders OLD-->;
@RN1644;<!-- Figuieredo 2015 16 sessions 49% increase young -->
@RN1809; <!-- Brook 18 sessions 8 and 40% (9 sessions) and 15,22% (18 sessions) old and young-->
@RN1897<!-- 11% 36 sessions young -->].
Time-course data is however sparse as only a limited number of studies have investigated exercise induced accumulation over multiple time-points.
Bickel *et al*. showed that two subsequent bouts of a hypertrophic stimulus increased total RNA to the highest levels at 72 h after a second bout of RT [@RN1520]<!-- Total RNA increase 72 h after a second bout -->. We have previously reported peak RNA levels 48 h after the fourth training session with a tendency towards normalization after 31 sessions
[@RN2358].
This is similar to what was reported by Brook *et al.* where peak values were seen after nine sessions followed by a slight decrease to after 18 sessions
[@RN1809].
These data suggests that ribosome accumulation, reaches a plateau in the early phase of RT.

Based on these observations we are hypothesizing that (1) ribosome accumulation occurs in the early phase (3-4 weeks) of RT and (2) reaches a plateau during this phase in response to constant volume RT and finally (3) that fluctuations in ribosome abundance occurs in response to altered training volume and training cessation. Subsequently the aim of this study is to characterize fluctuations in ribosomal biogenesis during the initial phase of RT, in response to altered training volumes and in response to a short-term de-training period.

# Methods

## Study overview
Eighteen volunteers were recruited to the study. Eligible participants had to be non-smokers, between 18 and 35 years of age, report a training history of less than one RT session per week during the six months leading up to the study and not consume dietary supplements or medication with known effects on muscle metabolism. Potential participants were further screened for injuries causing impaired strength and/or affecting their ability to perform RT, symptoms or history of disease or known adverse reactions to local anesthetics. Included participants were allocated to either an training group (TR, *n* = 11) or a non-training control group (CTRL, *n* = 8; see Table 1 for participant characteristics). TR performed a 12 session RT protocol consisting unilateral knee-extension to allow for within-participant comparison exercise volume. In TR, one leg performed constant volume (CONST, 6 sets per session) and the other leg performed variable volume over the course of the study (VAR, four sessions each with 6, 3 and 9 sets per session; Figure 1A). CTRL did not partake in RT and were instructed to continue their everyday activities. Muscle biopsies were sampled bilaterally in TR before and 48 h after the first session and 48 h after the fourth, fifth, eight, ninth and twelfth session as wells as after a seven day de-training period. Muscle biopsies were obtained from CTRL at three occasions with 48 h separating the first two and 2-4 weeks separating second and third sampling. TR and CTRL performed strength assessments at least seven days prior to the first biopsy sampling and 72 h after the twelfth session (EXP) and 24 h after the last biopsy (TR and CTRL). Appendicular lean mass (Dual-energy X-ray absorptiometry, DXA) and muscle thickness of *m. vastus lateralis* were assessed prior to the first biopsy (TR and CTRL) as wells as before the second to last (TR) and last (TR and CTRL) biopsy (Figure 1A).

## RT protocol
Prior to all RT-sessions, participants performed a standardized warm-up consisting of 5 min ergometer cycling (rating of perceived exertion (RPE): 12-14), followed by ten repetitions each of push-ups, sit-ups and back-extensions. After warm-up, participants performed unilateral knee-extension with the prescribed number of sets. Each set was prescribed with 10 repetitions maximum (RM). When sets was completed with either fewer (8) or more (12) repetitions than 10, the resistance was adjusted accordingly. Inter-set rest periods were 90 sec. Participants performed the first set alternating between the right and left leg every other session and the set of the contralateral leg was performed in the rest-period of the first leg. The second session of each four-session block (session 2, 6 and 10) were performed at a sub-maximal resistance (~90% of the previous session) with the same number of repetitions. In each session, after the knee-extension exercise, participants completed two sets each of three upper-body exercises (bench press, lateral pull-down and shoulder press).<!--Prior to each session, participants were asked to rate muscle soreness on a scale from 0 to 10 (0 = no soreness, 10 = hurting very bad, e.g. not being able to walk properly) [@Vickers2010]. After each session, participants were asked to rate their perceived exertion (Borg CR10).-->
After the completion of each session participants were given a standardized drink to aid recovery (0.15 g kg<sup>-1</sup> protein, 11.2 g kg<sup>-1</sup> carbohydrates and 0.5 g kg<sup>-1</sup> fat).


## Muscle strength, body composition and muscle thickness assessments
Muscle strength was assessed as maximal voluntary isokinetic (90&deg; sec<sup>-1</sup>) and isometric (60&deg; angle, fully extended leg 0&deg;) knee extension torque. After a brief warm-up (5-min cycling, RPE 12-14), participants were seated and secured in the individually adjusted dynamometer. Participants were instructed to gradually increase their effort during three warm-up repetitions (50, 60 and, 70% of subjective maximal effort). After a 30-sec restitution period participants were instructed to perform three repetitions with maximal effort in the concentric phase. Sixty seconds after the isokinetic test the lever automatically moved to a 60&deg; angle and participants were instructed to push against the lever enough to see feedback from the visual feedback system. After an additional 15-sec restitution period, participants were instructed to push against the lever with maximal effort. Within the same assessment session, participants remained seated in the dynamometer for measurement perfomed on both legs. The first measurement was alternated between legs every other session. For statistical treatment of the data, all successful attempts were used. The last strength assessment at baseline was performed at least seven days prior to the first biopsy sampling. At least one of the baseline strength tests was performed on separate day with two sessions allowed to be perform on the same day with a short rest between assessments. Post training assessments were performed 48 hours and eight days after the last session.

For determination of body composition participants were scanned using DXA (Lunar Prodigy densitometer, GE Healthcare, Madison, WI, USA) with the standard scanning mode (13-25 cm). Participants were lying supine within the scanning bed reference lines, with a strap secured around the ankles to ensure a standardized body position in each scan. The scans were conducted with participants in a fasted state between 07.00-10.00 AM, with empty bladder and wearing only under-wear. Prior to each scan, a phantom scan was run to prevent baseline drifting from affecting analyses. The same technician was used at each time point. Analyses was performed using GE enCORE version 17.0 software (GE Healthcare). Region of interest was customized for covering upper thigh, marked with a sqaure from pubic symphysis to lateral part of tuberculum major, and distal to art. genu.

Muscle thickness (MT) was measured using a B-mode ultra sound unit (SmartUS EXT-1M, Telemed, Vilnius, Lithuania). Participants lay supine in a relaxed position for 20 min before assessments, with their feet strapped in a standardized position. A mark was set on the line 60% of the distance between Spinia Iliac Anterior Superior and the lateral femur condyle. MT of *m.vastus lateralis* was measured applying a water-soluble transmission gel (Aquasonic 100 Ultrasound Transmission Gel; Parker Laboratories Inc., Fairfield, NJ, USA), and a 39 mm 12 MHz ultrasound probe was placed perpendicular to the site of interest without pressing the skin. When the quality of the image was satisfactory, evident as distinct upper and lower muscle fascia, three images were captured, where the probe was relocated to the same position between each image. Position of the probe was marked on the skin and subsequently marked on a transparent paper to ensure similar probe placement for both the right and left *m.vastus lateralis* at subsequent assessments. Analyses were done in ImageJ Fiji [@RN2561] with images cropped and coded to ensure blinding of the assessor.

## Muscle biopsy sampling 
Muscle specimens were sampled bilaterally from *m. vastus laterlis* under local anesthesia (Lidokain 10 mg ml<sup>-1</sup>, Mylan, Mylan Ireland Limited, Dublin, Ireland) using a disposable needle (12-14 gauge, Universal plus, Medax, Poggio Rusco, Italy), operated with a spring loaded device (Bard Magnum, Bard Norway, Rud, Norway). Two to four passes were made to get sufficient material. One to two aliquots of the samples were quickly dissected free from connective and fat tissue, weighed and frozen in isopentan chilled to -80&deg;C and stored at -80&deg;C until further processing.<!-- At baseline, after twelve weeks of training and, after the de-training period, a further aliquot from each sample was selected for immunohistochemistry analysis and subsequently immersed in 4% formalin for fixation.-->        

## RNA and protein extraction
Frozen muscle tissue was homogenized in 1 ml of Trizol (ThermoFisher Scientific, Oslo, Norway) spiked with 0.04 ng of an external, non-mammalian, RNA spike-in (Lambda PolyA External Standard Kit, Takara Bio Europe, Saint-Germain-en-Laye, France). Mechanical disruption of the samples was achieved using Zirconium Oxide Beads (0.5 mm, Next Advance, Inc., New York, USA) and a bead mill (Bullet blender, Next Advance). Chloroform (200 &#956;l) was added prior to centrifugation (12000 g, 15 min at 4&deg;C) to achieve phase separation. Four hundred fifty &#956;l of the upper aqueous phase was transferred to a fresh tube and 500 &#956;l of isopropanol was added to precipitate the RNA. After a 10 min incubation at room temperature, samples were centrifuged (12000 g, 10 min at 4&deg;C), after which a pellet formed. The pellet was washed three times in chilled 75% ethanol with centrifugation between each wash (7500 g, 5 min at 4&deg;C). After the final wash all ethanol was removed and the pellet was eluted in 0.1X Tris-EDTA buffer. RNA concentration and purity was assessed by spectrophotometry. All samples had 260/280 ratios > 1.95.

Protein was extracted from Trizol preparations according to manufacturers instructions and [@RN2050]<!-- Kopec trizol protocol --> with modifications. The remaining aqueous phase was removed and DNA was precipitated by the addition of 300 &#956;l of absolute ethanol followed by gentle centrifugation (2000 g, 5 min at room temperature). An aliquot of the phenol-ethanol phase, corresponding to ~1.75 mg of tissue, was transferred to to a fresh tube. After addition of at least two volumes of isopropanol and incubation (10 min at room temperature), samples were centrifuged (7500 g, 10 min 4&deg;C) and a pellet formed. The pellet was washed three times in 95% ethanol with each wash separated by centrifugation (5000g, 5 min at room temperature). After the last wash all liquid was removed and 45 &#956;l of Kopec buffer [@RN2050] was added (5% SDS, 10 mM Tris, 140 mM NaCl and 20 mM EDTA, pH 8; containing protease and phosphatase inhibitors). Pellets were incubated at 50&deg;C for three hours after which the majority of samples were dissolved. Any undissolved material was sedimented by centrifugation (10000 g, 10 min at room temperature). Protein concentrations were measured (Pierce Detergent Compatible Bradford Assay, ThermoFisher Scientific). Sample were normalized in 4X Laemmli buffer (Bio-Rad Norway AS, Oslo, Norway), boiled (95&deg;C, 5 min) and stored at -20&deg;C before later use.

## Quantitative polymerase chain reaction (qPCR)
Complementary DNA (cDNA) was synthesized in technical duplicates from 500 ng of total RNA using random hexemer and anchored Oligo-dT primers (Thermo Fisher Scientific) together with Superscript IV (Thermo Fisher Scientific) according to manufacturer's instruction. qPCR reactions were performed with diluted cDNA (2 &#956;l, 1:25 dilution), a SYBR-green based commercial master mix (PowerUp™ SYBR™ Green Master Mix, Thermo Fisher) and, target-specific primers (500 nM) in 10 &#956;l reaction volumes using a real-time detection system (QuantStudio 5 Real-Time PCR System, Thermo Fisher Scientific). Fast cycling was used (1 sec denaturing, 30 sec annealing) after UNG (2 min, 50&deg;C) and polymerase  (2 min, 95&deg;C) activation. Melt curves were collected from all reactions to confirm single product amplification. Primers were further evaluated by agarose gel electrophoresis which confirmed primer sizes and non-template control experiments confirming no amplification without template. Primer sequences and their respective average performances are shown in Table 2.
```{r check-primer-sizes, include = FALSE}

## To check primer sizes ##
# Be aware that the gel is skewed (overestimating sizes in last wells)

#
# mw <- read_excel("./data/tr010_primers.xlsx", sheet = "molecular_marker") %>%
#  mutate(logx = log(X_coord)) %>%
#   print()
# mw_primers <- read_excel("./data/tr010_primers.xlsx", sheet = "primers_mw") %>%
#     mutate(logx = log(X_coord)) %>%
#   print()
# 
# 
# m1 <- lm(MW ~ poly(logx,3), data = mw)
#
# mw %>%
#  ggplot(aes(logx, MW)) + 
#  geom_point() + 
#  geom_smooth(method = "lm", formula = y ~ poly(x, 3))
# 
# 
# mw_primers$mw <-predict(m1, newdata = mw_primers)
# 
# 
# library(ggrepel)
# read_excel("./data/tr010_primers.xlsx") %>%
#   mutate(predicted_product_size = rstart - fstart) %>%
#   dplyr::select(primer = primer_id, predicted_product_size) %>%
#   inner_join(mw_primers) %>%
#   ggplot(aes(predicted_product_size, mw, label = primer)) + 
#   geom_point() + 
#   ggrepel::geom_text_repel() + 
#   scale_x_continuous(limits = c(50, 160)) +
#   scale_y_continuous(limits = c(50, 160))+
#   geom_abline(slope = 1, intercept = 0)
# 



```
Raw fluorescence data was exported from the QuantStudio software and estimates of quantification cycle (Cq) and amplification efficiency was derived for each reaction using the qpcR package[@RN1768].

## Immunoblotting
Protein samples (20 &#956;g) were separated on 4-20% Tris-Glycin gels (Criterion TGX Precast Gels, Bio-Rad) at 250 V for 45 min using the recommended running buffer (25 mM Tris, 192 mM Glycin, 0.1% SDS). All samples from the same participant were run on the same gel and all samples were run in at least duplicates. Separated samples were transferred to &#956m PVDF membranes (Immun-Blot, Bio-Rad) using wet transfer (25 mM Tris, 192 Glycin, 10% vol/vol methanol) at a constant voltage of 300 mA for 3 h. Membranes were then stained to confirm transfer and enable total protein quantification using a reversible protein stain (Pierce Reversible Protein Stain, Thermo Fisher Scientiﬁc). Primary antibodies were acquired to detect UBF (F-9, sc-13125, Santa-Cruz Biotechnology, Dallas, Texas, USA) and ribosomal protein S6 (54D2, #2317, Cell Signaling Technology, Danvers, MA, USA). After blocking (Tris-buffered saline blocking buffer, 20 mM Tris, 150 mM NaCl, 5% fat-free milk, 0.1% Tween-20), membranes were incubated over-night with primary antibodies diluted in blocking buffer (UBF, 1:200; S6, 1:1000) followed by incubation with a secondary antibody conjugated to horseradish peroxidase (Anti-mouse IgG, #7076, Cell Signaling Technology, 1:10000). Membranes were washed 6 &#215; 5 min after incubation with primary antibodies and 8 &#215; 5 min after incubation with the secondary antibody. All incubation and washing steps were performed at 4&deg;C using an automatic membrane processor (BlotCycler, Precision Biosystems, Mansﬁeld, MA, USA). Chemiluminescent signals from membranes were detected after 5 min incubation in substrate (Super Signal West Femto Maximum Sensitivity Substrate, Thermo Fisher Scientiﬁc) using a documentation system. Total protein content was quantified from whole membrane images and defined as the mean gray value of the whole lane. Between-lane gray values were used as background subtracted from protein values. Total protein quantification was done using ImageJ Fiji [@RN2561]. Chemiluminescence signals were quantified using Image Studio Lite (LI-COR Biotechnology, Lincoln, NE, USA).

## Statistics and data analysis
Descriptive data are presented as mean and standard deviation (SD). The effect of training on muscle strength, muscle thickness, UBF and rpS6 protein, total RNA and gene abundances was assessed in mixed effects regression models with time and group (TR vs. CTRL) combined in a single factor as population (fixed) effects and leg nested within participant included as group level (random) effects. Data from TR that matched CTRL was included in the comparisons. Changes ($\Delta$) within each group were contrasted to each other (i.e. $\Delta$ TR - $\Delta$ CTRL) to estimate interactions. The effects of different volume conditions and general time-course patterns were assessed from data using all pairwise observations from the TR group. Based on graphical evaluation of protein and total RNA data, segmented regression models were used to estimate slopes in three segments (session 1-4, 4-8 and 8-12; corresponding to blocks of different volume prescription in the training group). When no robust effects of volume conditions were detected, group averages are presented. Segmented models were fitted with time and volume condition as population effects and legs nested within participants as group level effects.
Muscle strength, muscle thickness, protein and total RNA data was modeled after log transformation.
Gene abundance data was fitted with time as a categorical variable in comparing volume conditions and Cq values converted to counts as suggested by Matz et al. [@RN1964]. A Poisson-lognormal model was used to fit the count data using all genes in the data set and including group level effects for each technical duplicate controlling for technical errors in sample preparation. An offset consisting of a normalization factor proportional to the amount of muscle used to prepare cDNA was used to model gene abundance per tissue weight. The external reference gene was used to calculate the normalization factor (External reference counts $\times$ muscle weight (mg) in each Trizol preparation). The offset was specified as a predictor with the coefficient fixed to 1.

A linear model was used to estimate the increase per session and average total RNA for every leg in the TR group. These estimates were then used to estimate the effect of training induced increase per session and average total RNA abundance on muscle hypertrophy. For each leg, session was used as the independent variable centered on Session 6 and log transformed RNA per tissue weight as the dependent variable. Mean-centering of the independent variable was done to obtain an estimate of the average RNA concentration per leg. This also assured that the slope and intercept did not correlate, something that could lead to colinearity issues in subsequent modeling. A mixed effects model was subsequently fitted with differences in muscle thickness pre- to post-training as the dependent variable and estimated percentage per session increases in total RNA, the mean total RNA scaled as standard deviations from the mean and sex as independent variables. Leg nested within participant was used as group levels effects. The robustness of this model was assessed by leave-one-out analysis on the level of individual data points in the relationship between total RNA and sessions and on the level of participants (see Results).

All models were fitted in a Bayesian framework using either the brms [@RN2562] or MCMCglmm [@RN1992] package written for R [@Rcore]. Inference about effects of interest was done based on point estimates and their 95% credible intervals (CI). Credible intervals not containing null effects were interpreted as robust. Models were fitted with default priors which also makes CI analogous to confidence intervals but with the interpretation being that the CI contains the true population value with the specified certainty (95%), given the data.
Fitting performance was assessed by confirming convergence of at least four different chains of MCMC samples (graphically assessed and confirmed with $\hat{R}\approx 1$). Model performance was assessed from comparing simulated data from each model to observed data graphically (posterior predictive checks). 


# Results
```{r training-data}





m2 <- readRDS("./data/derivedData/training-data/m2.RDS")
# Model containing interaction 
# readRDS("./data/derivedData/training-data/m1.RDS")


# Create a hypothesis and convert to percentages


h <- hypothesis(m2, c("(weekW2 / Intercept) * 100 = 0",
                 "(weekW3 / Intercept) * 100 = 0"))$hypothesis 



load.inr.w2 <- paste0(sprintf("%.0f",h[1, 2]), 
                      "%, 95% CI: [",
                      sprintf("%.0f",h[1, 4]), 
                      ", ",
                      sprintf("%.0f",h[1, 5]), 
                      "]")
load.inr.w3 <- paste0(sprintf("%.0f",h[2, 2]), 
                      "%, [",
                      sprintf("%.0f",h[2, 4]), 
                      ", ",
                      sprintf("%.0f",h[2, 5]), 
                      "]")


```

All participants successfully completed their prescribed sessions. Different volume conditions resulted in diverging loading profiles (load &#215; repetitions) over the course of the study (Figure 1B). The intensity of exercise (load at 10RM) increased similarly in both conditions from the first to the second (`r load.inr.w2`) and third (`r load.inr.w3`) four-session block, respectively. Isokinetic strength increased to Session 12 in the experimental group compared to the non-training control group, with an attenuated difference seen after de-training (Figure 1C). Average isometric strength showed the same general pattern although with considerable more uncertainty indicated by wider 95% CI (Figure 1C). Muscle thickness of the *m. vastus lateralis* increased in the experimental group compared to the control group both after Session 12 and after eight days of de-training (Figure 1D). There were no indications of differences between training conditions in strength or muscle thickness changes.

```{r western-data}
segmented_protein_results <- readRDS("./data/derivedData/western-analysis/segmented_results.RDS")



s6_slope0_4 <- paste0(sprintf("%.1f", 100 * (exp(segmented_protein_results[6,4]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[6,6]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[6,7]) - 1)),
                   "]")
                      
s6_slope4_8 <- paste0(sprintf("%.1f", 100 * (exp(segmented_protein_results[7,4]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[7,6]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[7,7]) - 1)),
                   "]")
                      

s6_slope8_12 <- paste0(sprintf("%.1f", 100 * (exp(segmented_protein_results[8,4]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[8,6]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[8,7]) - 1)),
                   "]")

ubf_slope0_4 <- paste0(sprintf("%.1f", 100 * (exp(segmented_protein_results[14,4]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[14,6]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[14,7]) - 1)),
                   "]")
                      
ubf_slope4_8 <- paste0(sprintf("%.1f", 100 * (exp(segmented_protein_results[15,4]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[15,6]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[15,7]) - 1)),
                   "]")
                      

ubf_slope8_12 <- paste0(sprintf("%.1f", 100 * (exp(segmented_protein_results[16,4]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[16,6]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(segmented_protein_results[16,7]) - 1)),
                   "]")
                      

     

s6.m3 <- readRDS("./data/derivedData/western-analysis/s6_detrain.RDS")
ubf.m3 <- readRDS("./data/derivedData/western-analysis/ubf_detrain.RDS")



ubf.sum <- summary(ubf.m3)$fixed


var_s12 <- paste0(sprintf("%.1f", 100 * (exp(ubf.sum[3,1]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(ubf.sum[3,3]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(ubf.sum[3,4]) - 1)),
                   "]")

const_detrain <- paste0(sprintf("%.1f", 100 * (exp(ubf.sum[2,1]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(ubf.sum[2,3]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(ubf.sum[2,4]) - 1)),
                   "]")

var_const_interaction <- paste0(sprintf("%.1f", 100 * (exp(ubf.sum[4,1]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(ubf.sum[4,3]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(ubf.sum[4,4]) - 1)),
                   "]")

h <- hypothesis(ubf.m3, "condvar + timepost1w:condvar = 0")$hypothesis %>%
  data.frame()

var_change <- paste0(sprintf("%.1f", 100 * (exp(h[1,2]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(h[1,4]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(h[1,5]) - 1)),
                   "]")


```

Protein levels of RPS6 and UBF increased only after twelve sessions in the training group compared to control and stayed elevated after eight days of rest (Figure 2A and B). The average increase in protein levels over twelve sessions was near linear in both targets and estimated to `r s6_slope0_4`, `r s6_slope4_8` and `r s6_slope8_12` in RPS6, and `r ubf_slope0_4`, `r ubf_slope4_8` and `r ubf_slope8_12` in UBF per session between session 1-4, 4-8 and 8-12, respectively. Slopes did not differ between volume-conditions but there was a tendency towards lower levels of UBF in the variable condition at session 12 (`r var_s12`). After the de-training period the constant volume condition tended to show decreased levels of UBF (`r const_detrain`) while the variable volume condition remained at a similar level compare to session 12 (`r var_change`; interaction effect: `r var_const_interaction`). Levels of rpS6 did not change in the de-training period and were similar between volume conditions.    


```{r total-rna}

# Segmented model analysis in text 
# Read the segmented model 
tc.m2 <- readRDS("./data/derivedData/total-rna-analysis/tc.m2.RDS")


time1.hyp <- hypothesis(tc.m2, "time1 = 0")
time2.hyp <- hypothesis(tc.m2, c("time2 + time1 = 0")) # The slope (on log scale) between session 4 and 8
time3.hyp <- hypothesis(tc.m2, c("time2 + time1 +time3 = 0")) # The slope on log scale between session 8 and 12

slope0_4 <- paste0(sprintf("%.1f", 100 * (exp(time1.hyp$hypothesis[1,2]) - 1)),
                           "%, 95% CI: [",
                   sprintf("%.1f", 100 * (exp(time1.hyp$hypothesis[1,4]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(time1.hyp$hypothesis[1,5]) - 1)),
                   "]")
                      
slope4_8 <- paste0(sprintf("%.1f", 100 * (exp(time2.hyp$hypothesis[1,2]) - 1)),
                           "%, [",
                   sprintf("%.1f", 100 * (exp(time2.hyp$hypothesis[1,4]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(time2.hyp$hypothesis[1,5]) - 1)),
                   "]")
                      

slope8_12 <- paste0(sprintf("%.1f", 100 * (exp(time3.hyp$hypothesis[1,2]) - 1)),
                           "%, [",
                   sprintf("%.1f", 100 * (exp(time3.hyp$hypothesis[1,4]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(time3.hyp$hypothesis[1,5]) - 1)),
                   "]")
                      

## Effect of detraining 

detrain.hyp <- hypothesis(tc.m2, "detraindetrain = 0")

detrain_rna <- paste0(sprintf("%.1f", 100 * (exp(detrain.hyp$hypothesis[1,2]) - 1)),
                           "%, [",
                      sprintf("%.1f", 100 * (exp(detrain.hyp$hypothesis[1,4]) - 1)), 
                   ", ",
                       sprintf("%.1f", 100 * (exp(detrain.hyp$hypothesis[1,5]) - 1)),
                   "]")
                      



```

A single session of RT led to robust increase in 47S and 45S pre-rRNA expressed per unit tissue weight within the training group (Figure 3B) and compared to control (Figure 3C). These transcripts were also increased compared to control after twelve sessions together with the 45S ITS and mature rRNA species 28S and 18S (Figure 3C). After eight days of rest, only 18S and 28S remained elevated above levels measured in the control group (Figure 3B). This general pattern of rRNA expression was reflected also in total RNA expressed per unit tissue weight were a robust increase compared to control was seen after session twelve (Figure 3E and F). Within the experimental group, only small differences were seen in rRNA subspecies as 45S ETS was robustly higher after 12 sessions in the variable volume condition after an imbalance between conditions prior to training (Figure 3D). Total RNA increased similarly in both volume conditions throughout the training period (Figure 3G). Most of the increase was seen in the first four sessions where the average increase was `r slope0_4` per session in the intervention group as a whole. Between session four and eight, and eight to twelve changes in total RNA per tissue weight per session was not robustly different from zero (`r slope4_8` and `r slope8_12`). After the de-training period Total-RNA decreased by `r detrain_rna` from levels measured after Session 12.     

```{r combined-analysis}

# Save model 
comb.m1 <- summary(readRDS("./data/derivedData/ubf-tot-rna-model/comb_m1.RDS"))$fixed




ubf_cor_tot_rna <- paste0(sprintf("%.1f", 100 * (exp(comb.m1[2,1]) - 1)),
                           "% [",
                   sprintf("%.1f", 100 * (exp(comb.m1[2,3]) - 1)), 
                   ", ",
                   sprintf("%.1f", 100 * (exp(comb.m1[2,4]) - 1)),
                   "]")



```

UBF levels robustly predicted total RNA values as a standard deviation change in UBF levels led to a `r ubf_cor_tot_rna` increase in total RNA per unit tissue weight (Table X).
To estimate the influence of total RNA on muscle growth the average linear increase in total RNA was calculated for each leg together with the average total RNA abundance over the course of the training intervention in the training group.<!-- (estimated as the intercept in a regression model with number of sessions centered at session 6). move to methods -->
The relationship between muscle growth and characteristics of individual total RNA profiles were estimated in a regression model containing the increase in total RNA concentrations, expressed as percentage increase per session, and average total RNA abundance at Session 6, expressed as standard deviations from the mean. The model also contained sex as predictor to control for sex differences in muscle growth. Total RNA increase in response to training together with the average level of total RNA robustly predicted muscle growth (Table 4, Figure 4A and B). To assess the robustness of the model, each data point was iteratively removed from the data set and the model was refitted using new estimates of the total RNA abundance and increase. This analysis indicated that the relationship between Total RNA increase and muscle growth was preserved in all iterations but the upper limit of the 95% CI spanned 0 in some instances regarding the relationship between average RNA and muscle growth (Figure 4D). We further assessed the model by iteratively removing one participant from the data set, similarly this showed that estimates of the total RNA increase effect on muscle growth was robust but average total RNA estimates spanned 0 in some cases (Participant 8 and 3 in Figure 4D).


# Discussion

A main objective of this study was to establish a time course for training induced ribosomal biogenesis. We first set out to confirmed that markers of ribosome density were actually training induced and could do so as total RNA, ribosomal RNA subspecies and rpS6 mRNA and protein increased in the training group compared to the non-training control group. Total RNA has been used in many previous studies as an approximation of ribosomal density as the majority of total RNA is ribosomal [@RN2142]<!-- Figueriedo --> and total RNA concentrations predicts protein synthesis [@RN2145]<!-- Millward 1973 -->. Previous studies have indicated that the training induced rise in total RNA reaches a peak after which a plateau or decrease is observed
[@RN1809;<!-- brook 2016 -->
@RN2358].<!-- Hammarström -->
In the present study we observed a rapid increase in the first four sessions after which a plateau was observed. The observed plateau could be a consequence of the unit of measure as total RNA is estimated per unit tissue weight and that ribosome biogenesis occurs simultaneously as protein accretion leading to a dilution effect [@RN2142]<!-- Figueriedo  -->.
We could confirm muscle hypertrophy both after session 12 and after a period of de-training indicating that the dilution effect is a reasonable assumption in interpreting the observed plateau. Further supporting this view is UBF increased over the whole training period indicating that the transcriptional apparatus continued to increase over the whole course of the training period.  
From a mechanistic perspective, UBF is an important factor for rDNA transcription as it, in its active state recruits a secondary transcription factor (SL1) to the rDNA promoter and enables transcription by RNA polymerase I
[@RN2563].<!--Lin 2007 phospho of UBF shows it has a role in recruioting SL1 to the rDNA promoter -->
Activation of UBF is thought to be related to signaling through the mechanosensitive mTOR pathway as rapamycin, a specific mTOR inhibitor, blocks UBF from recruiting SL1 and subsequent rRNA transcription
[@RN1632; <!-- Nader rapamycin blocj UBF availability  -->
@RN2564]<!-- Hannan 2003 -->
<!-- Hannan 2003 10.1128/MCB.23.23.8862-8877.2003 mTOR is believed to play a role in phosphorylating UBF leading to recruitment of SL-1. The effect of blockage of mTOR through rapamycin is not rescued by hypophosphorylated UBF -->
Evidence from human exercise studies confirms training induced activation of UBF through phosphorylation
[@RN1746; <!-- Figueiredo 2016 cold water inhibits UBF activation and increase rRNA increase -->
@RN1644].<!-- Figueriedo ubf increase total and active form (ser388) --> 
In addition to exercise induced activation of UBF, mechanical loading leads to increased levels of total UBF
[@RN1746; <!-- Figueiredo 2016 cold water inhibits UBF activation and increase rRNA increase -->
@RN1644]. Increases in UBF was determined to be rapamycin insensitive after synergist ablation in mice
[@RN1072]<!-- Goodman 2011 UBF increase due to synergist ablation is rapamycin insensitive https://doi.org/10.1113/jphysiol.2011.218255 --> 
pointing to an effect observed in cell models where c-Myc induces UBF mRNA transcription [@RN2565].<!-- Pootinga 2004 MAD1 and c-Myc controls UBF transcription UBF is required for c-Myc induced rDNA transcription -->
Interestingly the avalability of UBF *per se* has been shown to regulate rRNA transcription
[@RN2566]<!-- Hannan 1996 over expression of UBF leads to rRNA transcription in cardiomyocytes -->  through control of rDNA gene activity
[@RN2556].<!-- Sanij, UBF depleting leads to inactivation of rDNA genes -->
In the present study we measured total UBF levels and observed that the relative amount of UBF was related to total RNA concentrations over the whole course of the study indicating its role in ribosomal biogenesis. Together the above suggests that rRNA transcription may not plateau during short term training but muscle hyertrophy leads to a dilution effect. This observation has consequences for the use of total RNA per unit tissue weight as a determinant muscle growth as the muscle growth itself affects the measurement.

After the short de-training period, total RNA levels decreased in the training group (check if this is correct robust? More than no change in control?). This occured at the same time as no additional muscle hypertrophy was observed. [To do: check de training effect on pre-rRNA and total rRNA].
<!-- This is in contrast to what was concluded from a recent study of electrical stimulation in rats [@RN2567]. Comparing the time course in the present to that from electrical stimulation in rats highlights that regulation and time course of ribosome biogenesis in response to mechanical stress may diffe between species.-->
Resistance training volume, expressed in human exercise studies as the number of sets performed per session and muscle group is known to be a potent modulator of molecular mechanisms determining protein synthesis and ribosomal biogenesis including c-Myc induction, mTOR activation
[@RN2358; <!-- Hammarström volume affects ribo biogenesis and mTOR and c-myc -->
@RN1837; <!-- Ahtianinen 2015 volume affects mTOR differently -->
@RN791]<!-- Burd ex volume protein synthesis and anabolic [rpS6] phospho signalling-->,
total RNA increases [@RN2358] and post exercise protein synthesis [@RN791]
and subsequent training outcomes [@RN1767; @RN2358].<!-- Schoenfeld 2016 meta and Hammarström -->
We have previously shown different amounts of training volume in constant volume protocols leads to differences in total RNA and rRNA accumulation in the initial phase of RT [@RN2358]. Based on the notion that ribosomal biogenesis is a complex tightly regulated and energy demanding process for any cell [@RN1820] we hypnotized that fluctuations in training load would reflect in markers of ribosomal biosynthesis reflecting the current need. Comparing the two volume conditions in the present study we only found one part of the 45S ETS to be differentially expressed at Session 12 in favor of VAR. Together with a tendency of a rescued UBF level in response to increased volume in the VAR protocol after de-training, these observations do not give much support for this hypothesis. It is instead possible that after the initial four sessions a reduction of training load was sufficient to maximize ribosome biogenesis in this short time span. This leads us to suggest that ribsomal biogenesis is not sensitive to fluctuations in training volume during the initial phase of resistance training.

In conclusion...q



<!-- UBF and S6 protein levels may be differently regulated in men and rats as more consecutive bouts of electrical stimulation in rats does not prtoduce the same pattern seen in our study 
 Kotani 2019 10.1152/japplphysiol.00665.2018 increase in t UBF at 6 h post ex but not 48 despite 2 weeks training (3 x week). S6 increase at 48 h and 6 bouts but return to sed values at 18 bouts. 18 and 28 S increased over sed at 48 h post ex with no further increase beyond 3 bouts 45S maximal at three bouts -->







# References
<div id="refs"></div>

# Figure legends

**Figure 1.** Study design (A) training load (B) and training outcomes (muscle strength, C; muscle thickness D).   


**Figure 2.**  Total RNA and ribosomal RNA subspecies in response to trainin. Total RNA increased compared to non-training controls to post-training (12 sessions) and tended to normalize after de-training (A). There were no robust differences between different volume conditions in the training group (A). B shows primer locations targeting different ribosomal RNA subspecies. Subsets of ribosomal RNA showed robust increases compared to control 48 h after the first session and other after the training period (C). Error bars shows 95% CI. Asterisk in C indicates robust differences between volume conditions (a 95% CI of pairwise differences not containing 0).    


**Figure 3.** mRNA (A and B) and protein abundances (C and D) of ribosomal protein S6 and Upstream bindning factor (UBF). mRNA abundaces are expressed per total RNA (round points) or per muscle weight (triangles). Black points and error bars represents statistically robust results (a 95% CI not containing 0). E shows western-blots and total protein stains from a representative participant.   
